internal/db/db.go

```diff
@@ -0,0 +1,84 @@
+package db
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+)
+
+type ChatInfo struct {
+	ChatID int64
```

Comment:
maybe not in this PR but we can set like a time for each chat so not all groups be notified at the same time

Reply:
I agree, we can add this as new feature.